/mir_teleop_joypad:
    ros__parameters:
        joypad:
            buttons:
                name: [deadman, run, base_motors_on_off, arm_motors_on_off, reconnect_left, reconnect_right, arm_motor_1_2, arm_motor_3_4, arm_motor_5, arm_cart_ctrl, gripper, print_arm_joint_states]
                index: [5, 7, 8, 9, 10, 11, 0, 1, 2, 4, 3, 6]
            axes:
                name: [base_linear_x, base_linear_y, base_angular_z, arm_linear_x, arm_linear_y, arm_linear_z, arm_angular_x, arm_angular_y, arm_angular_z, arm_joint_axes_1, arm_joint_axes_2]
                index: [1, 0, 2, 1, 0, 3, 2, 5, 4, 4, 5]
        base_max_linear_x_vel: 0.3          # m per second in run mode
        base_max_linear_y_vel: 0.3          # m per second in run mode
        base_max_angular_vel: 0.6           # rad per second in run mode
        arm_max_vel: 0.4                    # rad per second in run mode
        soft_joint_limit_threshold: 0.1     # rad
